openpgp/packet/symmetrically_encrypted_test.go

```diff
@@ -28,17 +31,21 @@ func (t *testReader) Read(buf []byte) (n int, err error) {
 	if n > len(buf) {
 		n = len(buf)
 	}
-	copy(buf, t.data)
+
+	copy(buf[:n], t.data)
 	t.data = t.data[n:]
+
 	if len(t.data) == 0 {
 		err = io.EOF
 	}
+
 	return
 }
 
-func testMDCReader(t *testing.T) {
```

Comment:
This function (uncapitalized) was not used. Fixed it, but also had to adjust the test data, as it was broken

Reply:
Rip. Alrighty, thanks :+1: